SignalShareExtension/ShareViewController.swift

```diff
@@ -295,9 +304,15 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate, S
     }
 
     private func showErrorView(title: String, message: String) {
+        // ensure view is visible.
+        self.view.layer.removeAllAnimations()
+        UIView.animate(withDuration: 0.1, delay: 0, options: [.curveEaseInOut], animations: {
```

Comment:
Why fade in the error view?  Why not just "snap" it in?

Reply:
Totally subjective - but to me snapping feels jarring. Is this a blocking concern?

Reply:
Okay. We snap in the recipient picker, right?  I don't understand why we'd snap in one and animate in the other.  Not a blocking concern though.

Reply:
No we animate it in from the bottom, as a modal.